docs/ops/filesystems/index.md

```diff
@@ -83,17 +101,21 @@ fs.hdfs.hadoopconf: /path/to/etc/hadoop
 This registers `/path/to/etc/hadoop` as Hadoop's configuration directory and is where Flink will look for the `core-site.xml` and `hdfs-site.xml` files.
 
 
-## Adding new File System Implementations
+## Adding a new pluggable File System implementation
 
-File systems are represented via the `org.apache.flink.core.fs.FileSystem` class, which captures the ways to access and modify files and objects in that file system. 
-Implementations are discovered by Flink through Java's service abstraction, making it easy to add new file system implementations.
```

Comment:
This looks like it still valid and could be kept as is imo.

Reply:
I've removed it because it was not exact:
 * it is not, that `FileSystem` classes are discovered by Java's service loading mechanism, but `FileSystemFactory`;
 * there is still reference left to Java's service loading in the `Add a service entry` step.

Reply:
Ok, given that this section is for developers of new filesystems I'm okay with the change.